backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConnectorAsUsageImpl.java

```diff
@@ -289,10 +289,11 @@ public EList<Association> getAssociation() {
      */
     @Override
     public EList<Feature> getConnectorEnd() {
-        List<Element> endFeatures = this.getOwnedRelationship().stream()
-                .filter(EndFeatureMembership.class::isInstance)
-                .map(EndFeatureMembership.class::cast)
-                .flatMap(fm -> fm.getOwnedRelatedElement().stream())
+        List<Feature> endFeatures = this.getOwnedRelationship().stream()
```

Comment:
In some case, EndFeatureMembership are not use but instead normal FeatureMembership with feature with flag isEnd set to true like in :

```
connection networkConnections : DeviceConnection {
		end [1] part hub references networkHubs;
		end [0..4] part device references networkDevices;
	}
```